client/my-sites/plans/main.jsx

```diff
@@ -89,6 +89,7 @@ module.exports = React.createClass( {
 						sites={ this.props.sites }
 						plans={ this.props.plans.get() }
 						siteSpecificPlansDetailsList={ this.props.siteSpecificPlansDetailsList }
+						enableFreeTrials={ true }
```

Comment:
@Tug thanks for making this change.

Also, not necessarily something we need to change, but there is a shorthand for passing props that are true where you can omit the assignment:

```js
<PlansList enableFreeTrials />
```